lightwood/helpers/numeric.py

```diff
@@ -25,5 +26,5 @@ def can_be_nan_numeric(value):
     return isnan
 
 
-def filter_nan(series):
-    return [x for x in series if not can_be_nan_numeric(x)]
+def filter_nan_and_none(series: Iterable) -> list:
+    return [x for x in series if not can_be_nan_numeric(x) and x is not None]
```

Comment:
This is problematic; there is no guarantee for the length of the array to be preserved:

See following:
```
from lightwood.helpers.numeric import *
import numpy as np

def filter_nan_and_none(series) -> list:
    return [x for x in series if not can_be_nan_numeric(x) and x is not None]

x = [0, np.nan, 5, 1, 5, None, np.nan]

print("Original length of list", len(x))

y = filter_nan_and_none(x)
print("New Filtered length", len(y))
```